chainweb.cabal

```diff
@@ -370,7 +370,6 @@ library
         , process >= 1.5
         , random >= 1.1
         , random-bytestring >= 0.1
-        , resourcet >= 1.2
```

Comment:
Woah, nice dependency pruning!  Were these all unused?

Reply:
Yes. I just recently discovered the GHC flag: `-Wunused-packages`. Not sure in what GHC version it was introduced.